docusaurus/docs/React/components/message-components/ui-components.mdx

```diff
@@ -415,3 +418,46 @@ The side of the message list to render MML components.
 :::note
 `QuotedMessage` only consumes context and does not accept any optional props.
 :::
+
+## MessageBounceOptions
+
+This component is rendered in a modal dialog for messages that got bounced by the moderation rules.
+
+### children
+
+| Type      | Default                                                                 |
+| --------- | ----------------------------------------------------------------------- |
+| ReactNode | Localized string for "This message did not meet our content guidelines" |
+
+Use this prop to easily override the text displayed in the modal dialog for the bounced messages, without fully implementing a custom `MessageBounceOptions` component:
+
+```jsx
+import { MessageBounceOptions } from 'stream-react-chat';
+
+function MyCustomMessageBounceOptions(props) {
+  return <MessageBounceOptions {...props} text='My custom text' />;
```

Comment:
I cannot see the custom children passed to the component in this example.

Reply:
Oh, very good catch! Yes, it should be custom children instead of the `text` prop.